pkg/grammar/SyslParser.g4

```diff
@@ -65,7 +65,7 @@ var_in_curly    : CURLY_OPEN Name CURLY_CLOSE;
 query_var       : Name EQ (NativeDataTypes | name_str | var_in_curly) QN?;
 query_param     : QN query_var (AMP query_var)*;
 
-http_path_part :name_str | DIGITS;
+http_path_part :(name_str (QN (Name EQ Name (AMP Name EQ Name)*)+)?) | DIGITS;
```

Comment:
Nit: Just eyeballing it, the `+` seems redundant; any subsequent param pairs would be captured as `AMP Name EQ Name`. Also space at the start :).

```suggestion
http_path_part : (name_str (QN (Name EQ Name (AMP Name EQ Name)*))?) | DIGITS;
```